src/actinia_parallel_plugin/wsgi.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
```

Comment:
I think this file is not needed - only needed when this flask api should run as separate api from actinia_core